Project.toml

```
@@ -1,6 +1,6 @@
 name = "Nemo"
 uuid = "2edaba10-b0f1-5616-af89-8c11ac63239a"
-version = "0.45.3"
+version = "0.45.2"
```

```suggestion
version = "0.45.3"
```
You don't want to revert this